packages/components/src/components/carousel/carousel.ts

```diff
@@ -457,7 +457,7 @@ export default class SdCarousel extends SolidElement {
   }
 
   render() {
-    const { scrollController, slidesPerPage } = this;
+    const { scrollController, slidesPerMove } = this;
```

Comment:
@Vahid1919 was this a leftover from the last change or did we explicitly renamed it? I got a bit lost on the many changes to this behavior.

Reply:
@karlbaumhauer These are two different behaviours. SlidesPerPage determines the number of slides visible on the screen at the same time. SlidesPerMove determines the amount of slides a carousel does per click.

I still have to closely test how these changes affect the behaviour but a lot of these tests are failing at moment so they should be resolved first.

Reply:
I understand the difference but I am just not sure with what we ended up in the last of the 100 dicussions on how we want to build it... ;-)

Would be awesome if you can have a closer look into it.

Reply:
@karlbaumhauer 
I updated the dot navigation logic by replacing slidesPerPage with slidesPerMove to correctly align navigation when slidesPerPage is greater than 1.

When slidesPerPage was greater than 1, clicking on a dot caused the carousel to jump to an incorrect slide. This happened because the dot navigation used slidesPerPage in the calculation, leading to unexpected results, like skipping slides or misaligned slide positions.

To fix this, I replaced slidesPerPage with slidesPerMove in the dot navigation calculations. Now, each dot click targets index * slidesPerMove, ensuring that the carousel accurately aligns with the intended set of slides.

Ticket: https://github.com/solid-design-system/solid/issues/1398

Reply:
ah ok. Thx for adding context.